src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -114,28 +119,102 @@ static void checkArg(string arg, string name)
                 Arguments = appArgs,
                 WorkingDirectory = workingDir,
                 GuiApp = guiApp,
-                EnvironmentVariables = envVariables
+                EnvironmentVariables = envVariables,
+                IsRunWithExit = metricType == MetricType.DotnetWatch ? false : true
```

Comment:
I wonder if this should not key off of `DotnetWatch` and instead just be a command line argument. You could imagine other things that might want this behavior.

Reply:
That is probably the right approach here. We can just include it as a test trait that is set, and also add it as a command line argument to startup.